strongbox-storage/strongbox-storage-core/src/main/java/org/carlspring/strongbox/configuration/Configuration.java

```diff
@@ -136,6 +139,10 @@ public int getPort()
         return port;
     }
 
+    public Set<String> getDigestAlgorithmSet() {
+        return digestAlgorithmSet;
+    }
```

Comment:
Cosmetic: Braces on new lines, please.

```suggestion
    public Set<String> getDigestAlgorithmSet()
    {
        return digestAlgorithmSet;
    }
```